### PR TITLE
Documentation Chatbot

### PR DESCRIPTION
**Pull request by jordanrburger**
Jira issue(s): N/A

Added the beta version of our documentation chatbot running via DialogFlow

Changes:

-

---

### CODE REVIEW

**Comment by jordanrburger** [comment]
Hey @pavelsuraba! here's the PR

**Comment by pavelsuraba** [comment]
> Hey @pavelsuraba! here's the PR

I've added some css changes to make chat looks better!

File: _layouts/main.html [hunk 1]
```diff
@@ -100,5 +100,35 @@ <h1>{{ page.title }}</h1>
...
+		--df-messenger-chat-window-box-shadow: 0px 0px 59px 0px rgba(0, 0, 0, 0.25);
+		--df-messenger-titlebar-title-font-size: 16px;
+		--df-messenger-send-icon-color: #a2aab8;
+		--df-messenger-send-icon-color-active: #0B57D0;
+		--df-messenger-send-icon-color-hover: #0B57D0;
+		--df-messenger-input-padding: 8px 0 8px 16px;
+ 	}
+</style>
+<!-- End KAI chatbot -->
+</body>
```
Comment (ondrejhlavacek) [inline _layouts/main.html:131]: duplicate body and html closing tags

**Comment by jordanrburger** [comment]
It has been approved in writing here: https://keboolaglobal.slack.com/archives/C050ZLP5P19/p1699691822165889?thread_ts=1699630651.092599&cid=C050ZLP5P19

File: _layouts/main.html (hunk 1 above)
Comment (jordanrburger) [reply _layouts/main.html:131]: Removed the duplicates

**Comment by jordanrburger** [comment]
Hey @ondrejhlavacek - when you have a moment, would you mind doing one more review?

**Review by ondrejhlavacek — APPROVED** [review]
looks good to me

**Comment by ondrejhlavacek** [comment]
Sure, do I need to check anything after the merge or can you do it?

**Comment by jordanrburger** [comment]
I'll take care of it! Just need to confirm that this chat widget is showing up in the lower right (and that logs are being captured in GCP)
Screenshot 2023-11-13 at 4 37 44 PM

**Pull request merged**